code-challenges.test.js

```diff
@@ -34,9 +34,67 @@ const object2 = { number: 0 }
 const object3 = { number: -7 }
 // Expected output: "-7 is not divisible by three"
 
-// b) Create the function that makes the test pass.
+describe ("evenDivisible", () => {
+    it("takes a object as an argument and decides if the number inside it is evenly divisible by three or not.", () => {
+        expect(evenDivisible(15)).toEqual("15 is divisible by three")
+        expect(evenDivisible(0)).toEqual("0 is divisible by three")
+        expect(evenDivisible(-7)).toEqual("-7 is not divisible by three")
+
+    })
+}) // Failed succesfully: output: ReferenceError: evenDivisible is not defined
 
+// b) Create the function that makes the test pass.
+// -------------- FIRST ATTEMPT AND IT DIDNT WORK: THE TEST CONTINUES TO SAY THAT evenDivisible is not defined---------------
```

Comment:
This is because you are adding a method into the test variable, which does 2 things: It alters the original test variable, so when your test looks for object1, it now sees an object with 2 things in it, and also the method only exists in the scope of the object, so when your test looks to perform a function call on `evenDivisible`, it can't see it anywhere since it lives in the scope of your object, so it returns that it is not defined.